bin/desi_submit_prod

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+# coding: utf-8
```

Comment:
detail: I think the proper syntax for this is
```
# -*- coding: utf-8 -*-
```
but I'm also pretty sure that that is a leftover from python 2 times, and utf-8 is the default for python 3 so this isn't needed anymore.

Reply:
This was added automatically by my IDE. It's present in several other scripts we use in the pipeline and doesn't seem to be harming anything, but I've replaced it with the syntax you recommend.